src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -222,6 +223,18 @@ def remove_snapshot(self, snapname):
         snappath = self.snapshot_path(snapname)
         rmsnap(self.fs, snappath)
 
+    def snapshot_info(self, snapname):
+        snappath = self.snapshot_path(snapname)
+        try:
+            btime = self.fs.getxattr(snappath, 'ceph.snap.btime')
+            rbytes = self.fs.getxattr(snappath, 'ceph.dir.rbytes')
+            data_pool = self.fs.getxattr(snappath, 'ceph.dir.layout.pool').decode('utf-8')
+        except cephfs.Error as e:
+            raise VolumeException(-e.args[0], e.args[1])
+        return {'size': int(rbytes), 'created_at': str(datetime.fromtimestamp(float(btime))), 'data_pool': data_pool,
```

Comment:
minor nit: better to return from try block?

Reply:
Done